src/Sidebar/SidebarDrawer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTheme } from '@material-ui/core/styles';
```

Comment:
@tjdalsans  you need to import useStyles here.

Reply:
It fixed the issue thank you!